.github/workflows/bash-matrix.yml

```
@@ -0,0 +1,94 @@
+name: Bash Matrix CI
+
+on:
+  push:
+    branches: [ main, trunk, master ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)